plugins/kubernetes-backend/src/service/KubernetesFanOutHandler.ts

```diff
@@ -336,6 +336,9 @@ export class KubernetesFanOutHandler {
     if (clusterDetails.dashboardParameters) {
       objects.cluster.dashboardParameters = clusterDetails.dashboardParameters;
     }
+    if (clusterDetails.displayName) {
+      objects.cluster.displayName = clusterDetails.displayName;
```

Comment:
if is not really necessary here. You could simplify this and just assign the value.